kong/plugins/hmac-auth/access.lua

```diff
@@ -82,11 +87,26 @@ local function create_hash(request, hmac_params, headers)
   return ngx_sha1(hmac_params.secret, signing_string)
 end
 
+local function is_signature_equal(signature_1, signature_2)
```

Comment:
what is the purpose of this function? constant time comparison? if so, please see 7345134740dbb01057b9d69f1f8bc91019a20378 as to why this approach is incorrect; otherwise, please elaborate :)